cclient/response/get_neighbors.h

```diff
@@ -12,31 +12,46 @@
 extern "C" {
 #endif
 
+#include "types/types.h"
+
 typedef struct {
   /**
-   * Address of your peer.
+   * URI of your peer.
    */
-  address_t address;
+  char_buffer_t* address;
```

Comment:
Maybe rename it `uri` as well ?

Reply:
not sure if rename is a good idea, here is keeping the same name in IRI response.

Reply:
You're right !